packages/http-client-csharp/emitter/src/lib/converter.ts

```diff
@@ -63,7 +64,7 @@ export function fromSdkType(
   if (sdkType.kind === "utcDateTime" || sdkType.kind === "offsetDateTime")
     return fromSdkDateTimeType(sdkType);
   if (sdkType.kind === "duration") return fromSdkDurationType(sdkType as SdkDurationType);
-  if (sdkType.kind === "tuple") return fromTupleType();
+  if (sdkType.kind === "tuple") return fromTupleType(sdkType as SdkTupleType);
```

Comment:
we do not need `as` here.
```suggestion
  if (sdkType.kind === "tuple") return fromTupleType(sdkType);
```